ui_framework/components/tool_bar/tool_bar_search_box.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, mount } from 'enzyme';
+import sinon from 'sinon';
+import { commonHtmlProps } from '../../test/common_html_props';
+
+import {
+  KuiToolBarSearchBox,
+} from './tool_bar_search_box';
+
+describe('KuiToolBarSearchBox', () => {
+  const onFilter = sinon.spy();
+
+  test('renders KuiToolBarSearchBox', () => {
+    const component = <KuiToolBarSearchBox onFilter={onFilter} { ...commonHtmlProps } />;
+    expect(render(component)).toMatchSnapshot();
+  });
+
+  describe('onFilter', () => {
+    const searchBox = mount(<KuiToolBarSearchBox onFilter={onFilter} { ...commonHtmlProps } />);
+    onFilter.reset();
+
+    test('is called on change event', () => {
+      const event = { target: { value: 'a' } };
+      searchBox.find('input').simulate('change', event);
+      sinon.assert.calledOnce(onFilter);
+    });
+
+    test('is called with the value entered', () => {
+      sinon.assert.calledWith(onFilter, 'a');
+    });
```

Comment:
Well I suppose your version would pass if it was called twice with 'a'?  But I don't have a problem combining.